### PR TITLE
Added tutorial for template literals in the existing string tutorial

### LINKED ISSUE #58: Add tutorial for Template Literals in Data types.
**Description of the tutorial**
In the current tutorial for strings, template literals are not mentioned. I feel template literals would be a much needed addition to the strings' tutorial.

**Do you want to be assigned to work on this?**

- [x] YES
- [ ] NO

### PR DESCRIPTION
**Pull request by bigsbunny**
### 2. Description
I have added a tutorial for Template Literals in Javascript to the existing tutorial on Strings. A challenge related to Template Literals has been added at the end as well to test the understanding of the reader.

### 3. Fill in checklist by marking [x]

- [x] I've read the CONTRIBUTING.md
- [x] I was assigned to this issue
- [x] My code is formatted using required linting
- [x] I've run my code locally and checked it works

Kindly consider this PR as a contribution for Hacktoberfest!

### CODE REVIEW

File: 2-variables/strings.js [hunk 1]
```diff
@@ -37,6 +37,42 @@ console.log(userName[2]); // will print "u"
...
+    - The benefit of using Template literals are that, it allows us to embed variables & expressions inside the string.
+      - This can be done using wrapping the expression inside of ${...} in the string.
+      - The value returned by the expression will then become a part of the string.
+      - TODO - After you learn about the various types of expressions in Javascript, try embedding the various types in template strings to see how they work.
+*/
+
+let userFullName = "Niles Maxwell";
+let numberOfTasksToday = 5;
+
+console.log(`Greetings ${userName}, you have ${numberOfTasksToday} tasks to be done today.`); // Logs "Greetings Niles, you have 5 tasks to be done today."
```
Comment (sabderemane) [inline 2-variables/strings.js:55]: Here it's using `userName` variable, so the comment is wrong, it's "Greetings sourabh, ..." You should change the comment and the variable used or remove the variable `userFullName` and change the name in the comment

**Review by sabderemane — CHANGES_REQUESTED** [review]
Nice work @bigsbunny ! Just a little change needed and it will be fine

File: 2-variables/strings.js (hunk 1 above)
Comment (bigsbunny) [reply 2-variables/strings.js:55]: @sabderemane apologies for that. Will fix it right now. Thank you for pointing it out!

**Review by sabderemane — APPROVED** [review]

**Comment by sabderemane** [comment]
Thanks for your contribution @bigsbunny 🎉

**Pull request merged**